packages/babel-plugin-transform-inline-environment-variables/__tests__/inline-env-var-test.js

```diff
@@ -20,4 +20,28 @@ describe("inline-env-plugin", () => {
 
     process.env.NODE_ENV = prev;
   });
+
+  it("should only include whitelisted variables if include option is specified", () => {
+    const prev = process.env.NODE_ENV;
+    process.env.NODE_ENV = "development";
+    const include = ["IS_ELECTRON"];
+
+    const source = "process.env.NODE_ENV";
+    const expected = "process.env.NODE_ENV;";
+    expect(transform(source, { include })).toBe(expected);
```

Comment:
The test is testing that only variables that are specified in the `include` param get transformed. Since `IS_ELECTRON` is the only variable specified in `include`, `process.env.NODE_ENV` should _not_ be transformed.

I'm not exactly sure what you mean by adding `process.env.IS_ELECTRON` to the source, but I'm interpreting it as make sure that it gets transformed?